pkg/ddl/ingest/env.go

```diff
@@ -29,92 +29,80 @@ import (
 )
 
 var (
+	litLogger = logutil.BgLogger().With(zap.String(logutil.LogFieldCategory, "ddl-ingest"))
```

Comment:
This PR has bug, when program `init`, `logutil.BgLogger()` is not set yet

Reply:
So, you will merge the log fix pr first and merge two prs in one cp pr?

Reply:
I'll reopen this PR, too much conflict